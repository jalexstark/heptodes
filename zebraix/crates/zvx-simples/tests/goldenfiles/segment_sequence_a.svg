<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="200pt" height="150pt" viewBox="0 0 200 150">
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 9.089844 134.195312 L 9.089844 15.804688 L 190.910156 15.804688 L 190.910156 134.195312 Z M 9.089844 134.195312 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 148.625 104.597656 C 212.050781 104.597656 148.625 41.171875 148.625 104.597656 Z M 148.625 104.597656 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 157.082031 66.542969 C 220.507812 66.542969 157.082031 3.117188 157.082031 66.542969 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 97.886719 121.511719 C 97.886719 79.226562 140.167969 79.226562 140.167969 121.511719 Z M 97.886719 121.511719 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 106.34375 58.085938 C 106.34375 15.804688 148.625 15.804688 148.625 58.085938 L 106.34375 58.085938 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 34.460938 125.738281 C 38.167969 123.328125 41.917969 119.433594 44.746094 115.054688 C 47.570312 110.675781 49.191406 106.242188 49.191406 102.90625 C 49.191406 99.570312 47.570312 97.652344 44.746094 97.652344 C 41.917969 97.652344 38.167969 99.570312 34.460938 102.90625 C 30.753906 99.570312 27.003906 97.652344 24.175781 97.652344 C 21.351562 97.652344 19.730469 99.570312 19.730469 102.90625 C 19.730469 106.242188 21.351562 110.675781 24.175781 115.054688 C 27.003906 119.433594 30.753906 123.328125 34.460938 125.738281 Z M 34.460938 125.738281 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 72.515625 125.738281 C 76.222656 123.328125 79.972656 119.433594 82.800781 115.054688 C 85.625 110.675781 87.246094 106.242188 87.246094 102.90625 C 87.246094 99.570312 85.625 97.652344 82.800781 97.652344 C 79.972656 97.652344 76.222656 99.570312 72.515625 102.90625 C 68.808594 99.570312 65.058594 97.652344 62.230469 97.652344 C 59.40625 97.652344 57.785156 99.570312 57.785156 102.90625 C 57.785156 106.242188 59.40625 110.675781 62.230469 115.054688 C 65.058594 119.433594 68.808594 123.328125 72.515625 125.738281 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 21.777344 28.488281 L 93.65625 50.054688 L 93.65625 28.488281 L 21.777344 28.488281 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 93.65625 57.242188 L 21.777344 35.675781 L 21.777344 57.242188 Z M 93.65625 57.242188 "/>
</svg>
