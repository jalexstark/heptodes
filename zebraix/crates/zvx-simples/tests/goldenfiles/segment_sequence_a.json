{
  "sizing_scheme": "SquareCenter",
  "base_width": 200.0,
  "base_height": 150.0,
  "axes_range": [
    -1.2,
    -1.4,
    3.1,
    1.4
  ],
  "padding": [
    0.05
  ],
  "drawables": [
    {
      "drawable": {
        "SegmentSequence": {
          "completion": "Closed",
          "segments": [
            {
              "Polyline": [
                [
                  -1.2,
                  -1.4
                ],
                [
                  -1.2,
                  1.4
                ],
                [
                  3.1,
                  1.4
                ],
                [
                  3.1,
                  -1.4
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "SegmentSequence": {
          "completion": "Closed",
          "path_choices": {
            "color": "Blue"
          },
          "segments": [
            {
              "Cubic": [
                [
                  2.1,
                  -0.7
                ],
                [
                  3.6,
                  -0.7
                ],
                [
                  2.1,
                  0.8
                ],
                [
                  2.1,
                  -0.7
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "SegmentSequence": {
          "path_choices": {
            "color": "Green"
          },
          "segments": [
            {
              "Cubic": [
                [
                  2.3,
                  0.2
                ],
                [
                  3.8,
                  0.2
                ],
                [
                  2.3,
                  1.7
                ],
                [
                  2.3,
                  0.2
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "SegmentSequence": {
          "completion": "Closed",
          "path_choices": {
            "color": "Blue"
          },
          "segments": [
            {
              "Cubic": [
                [
                  0.9,
                  -1.1
                ],
                [
                  0.9,
                  -0.10000000000000009
                ],
                [
                  1.9,
                  -0.10000000000000009
                ],
                [
                  1.9,
                  -1.1
                ]
              ]
            },
            {
              "Polyline": [
                [
                  1.9,
                  -1.1
                ],
                [
                  0.9,
                  -1.1
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "SegmentSequence": {
          "path_choices": {
            "color": "Green"
          },
          "segments": [
            {
              "Cubic": [
                [
                  1.1,
                  0.4
                ],
                [
                  1.1,
                  1.4
                ],
                [
                  2.1,
                  1.4
                ],
                [
                  2.1,
                  0.4
                ]
              ]
            },
            {
              "Polyline": [
                [
                  2.1,
                  0.4
                ],
                [
                  1.1,
                  0.4
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "SegmentSequence": {
          "completion": "Closed",
          "path_choices": {
            "color": "Blue"
          },
          "segments": [
            {
              "Arc": {
                "angle_range": [
                  -1.7101356675526658,
                  1.7101356675526658
                ],
                "center": [
                  -0.557513768686074,
                  -0.8970731707317072
                ],
                "transform": [
                  0.30590086546026746,
                  0.23707317073170747,
                  -6.965282253322208e-17,
                  0.2726424587382621
                ]
              }
            },
            {
              "Arc": {
                "angle_range": [
                  -1.710135667552666,
                  1.710135667552666
                ],
                "center": [
                  -0.6424862313139259,
                  -0.8970731707317072
                ],
                "transform": [
                  -0.3059008654602676,
                  0.23707317073170747,
                  0.0,
                  -0.27264245873826215
                ]
              }
            }
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "SegmentSequence": {
          "path_choices": {
            "color": "Green"
          },
          "segments": [
            {
              "Arc": {
                "angle_range": [
                  -1.710135667552666,
                  1.710135667552666
                ],
                "center": [
                  0.34248623131392597,
                  -0.8970731707317072
                ],
                "transform": [
                  0.3059008654602676,
                  0.23707317073170736,
                  -1.1937518019851725e-18,
                  0.27264245873826215
                ]
              }
            },
            {
              "Arc": {
                "angle_range": [
                  -1.7101356675526658,
                  1.7101356675526658
                ],
                "center": [
                  0.25751376868607406,
                  -0.8970731707317072
                ],
                "transform": [
                  -0.30590086546026757,
                  0.23707317073170747,
                  0.0,
                  -0.2726424587382621
                ]
              }
            }
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "SegmentSequence": {
          "path_choices": {
            "color": "Green"
          },
          "segments": [
            {
              "Polyline": [
                [
                  -0.9,
                  1.1
                ],
                [
                  0.7999999999999999,
                  0.5900000000000001
                ],
                [
                  0.7999999999999999,
                  1.1
                ],
                [
                  -0.9,
                  1.1
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "SegmentSequence": {
          "completion": "Closed",
          "path_choices": {
            "color": "Blue"
          },
          "segments": [
            {
              "Polyline": [
                [
                  0.7999999999999999,
                  0.42000000000000004
                ],
                [
                  -0.9,
                  0.93
                ],
                [
                  -0.9,
                  0.42000000000000004
                ],
                [
                  0.7999999999999999,
                  0.42000000000000004
                ]
              ]
            }
          ]
        }
      }
    }
  ]
}
